tools/small_rna_maps/small_rna_maps.xml

```diff
@@ -86,15 +87,15 @@
   ]]></command>
  <inputs>
     <conditional name="inputs_cond">
-        <param name="normalize" type="boolean" label="Is there a need for normalization?" truevalue="yes" falsevalue="no" checked="false"/>
+        <param name="normalize" type="boolean" label="Will there be normalization?" truevalue="yes" falsevalue="no" checked="false"/>
         <when value="no">
             <param name="inputs" type="data" format="bam" label="Select multiple alignments to parse" multiple="True"/>
         </when>
         <when value="yes">
             <repeat name="series" title="Normalization factors" min="1">
-                <param name="inputs" type="data" format="bam" label="Select alignment to parse" />
+                <param name="inputs" type="data" format="bam" label="Select multiple alignments to parse" />
                 <param name="normalization" type="text" label="Enter size/normalization factors. Leave blank for no normalization (default)"
```

Comment:
This is weird: if leave blank for default, it means that blank is equivalent to "1". Then in the example you put '1', which is misleading because it may suggest that '1' is different of " ".
I suggest another approach: put 1 in the form as default value.